opentok/opentok-meet
# Update to newer opentok-whiteboard that has a fix for capture functionality

#### What is this PR doing?

Update to newer opentok-whiteboard that has a fix for capture functionality

#### How should this be manually tested?

Try using the capture button in the whiteboard in Chrome and make sure it downloads and image.
